modules/nf-core/sourmash/compare/meta.yml

```diff
@@ -0,0 +1,63 @@
+name: "sourmash_compare"
+description: Compare many FracMinHash signatures generated by sourmash sketch.
+keywords:
+  - compare
+  - FracMinHash sketch
+  - containment
+  - sourmash
+  - metagenomics
+  - genomics
+  - kmer
+tools:
+  - "sourmash":
+      description: "Compute and compare FracMinHash signatures for DNA and protein data sets."
+      homepage: "https://sourmash.readthedocs.io/"
+      documentation: "https://sourmash.readthedocs.io/"
+      tool_dev_url: "https://github.com/sourmash-bio/sourmash"
+      doi: "10.21105/joss.00027"
+      licence: "['BSD-3-clause']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - signatures:
+      type: file
+      description: Files containing signatures (hash sketches) of samples 
+      pattern: "*.{sig}"
```

Comment:
Is this a contradiction? The `meta` should be for one sample but `signatures` should be sketches for multiple samples?

Reply:
Ya i see how this is confusing. A signature represents a single sample, but inside the signature are multiple sketches (one for k=21, one for k=31, and one for k=51). It's a single file though because each sketch represents a the same sample and it's created in one pass so the data is only read in one time. `sourmash compare` will only use one of the sketches from the signatures file, and that's specified by the `--ksize` argument

Reply:
Awesome, makes perfect sense.